tests/PhpSpreadsheetTests/Reader/Html/HtmlTest.php

```diff
@@ -363,4 +364,46 @@ public function testBorderWithColspan(): void
             self::assertEquals(Border::BORDER_THIN, $border->getBorderStyle());
         }
     }
+
+    public function testDataType(): void
+    {
+        $html = '<table>
+                    <tr>
+                        <td data-type="b">1</td>
+                        <td data-type="s">1234567</td>
+                        <td data-type="f">=CONCAT("TEXT A ","TEXT B")</td>
```

Comment:
I would prefer to eliminate data-type in the formula statement. It should not be necessary. I would still like to retain the formula and the assertions based on it in order to confirm that existing behavior hasn't changed.